content/docs/concepts/content_negotiation.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Prometheus Scraping Protocol Content Negotiation
+sort_rank: 2
+---
+
+# Prometheus Scraping Protocol Content Negotiation
+
+## Abstract
+
+This document specifies the protocol negotiation mechanism used by Prometheus
+when scraping metrics from targets. It defines the Accept header format,
+supported Content Types, and the negotiation process for determining the best
+available format for metric exposition.
+
+## Introduction
+
+Prometheus supports multiple formats for scraping metrics, including both
+text-based and binary protobuf formats. Based on the value of the Accept header,
+Prometheus will pick the best available Content Type for its reply.
+
+## Protocol Types
+
+### Supported Protocols
+
+The following protocols are be supported by Prometheus:
+
+1. `PrometheusProto` - Binary protobuf format
+2. `PrometheusText0.0.4` - Prometheus text format version 0.0.4
+3. `PrometheusText1.0.0` - Prometheus text format version 1.0.0
+4. `OpenMetricsText0.0.1` - OpenMetrics text format version 0.0.1
+5. `OpenMetricsText1.0.0` - OpenMetrics text format version 1.0.0
```

Comment:
do we need this at all? I do find it useful to cover the currently-implemented options